src/ui/SearchAlerts/SearchAlerts.ts

```diff
@@ -287,55 +315,63 @@ export class SearchAlerts extends Component {
 
     let context: string;
     if (subscription.name) {
-      context = _.escape(subscription.name);
+      context = subscription.name;
```

Comment:
Because the subscription name contains HTML formatted facet selection.

Otherwise we end up with the manage panel showing something like `<span class='facet-selected'>(A, B , C)</span>`, stuff like that.

But thinking about it, this could be dangerous indeed, if the service was hacked or something. Could use that as stored XSS.

So I'm just gonna get the text content and strip the HTML tags for managing. It won't be as pretty, but that's less important.